docs/versioned_docs/version-2.50.0-LTS/widgets/icon.md

```diff
@@ -40,8 +40,8 @@ Following actions of the component can be controlled using the component specifi
 
 | <div style={{ width:"100px"}}> Actions </div> | <div style={{ width:"135px"}}> Description </div>                                                             | <div style={{ width:"135px"}}> How To Access </div>                                                              |
 | :-------------------------------------------- | :------------------------------------------------------------------------------------------------------------ | :--------------------------------------------------------------------------------------------------------------- |
-| setVisibility                                 | You can toggle the visibility of the Icon component via a component-specific action within any event handler. | Employ a RunJS query to execute component-specific actions such as `await components.icon1.setVisibility(false)`. |
-| click                                         | You can trigger the click action on Icon component via a component-specific action within any event handler.  | Employ a RunJS query to execute component-specific actions such as `await components.icon1.click()`.              |
+| setVisibility()                                 | Toggles the visibility of the Icon component. | Employ a RunJS query (for e.g., `await components.icon1.setVisibility(false))` or trigger it using an event. |
```

Comment:
In the description column, please use "Sets the visibility of the component." instead of "Toggles the visibility of the Icon component."